examples/spring-test/src/test/java/org/camunda/bpm/extension/process_test_coverage/examples/CoverageTestConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package org.camunda.bpm.extension.process_test_coverage.examples;
+
+import org.camunda.bpm.extension.process_test_coverage.spring_test.ProcessEngineCoverageProperties;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+
+@TestConfiguration
+public class CoverageTestConfiguration {
+
+    @Bean
+    public ProcessEngineCoverageProperties processEngineCoverageProperties() {
+        return new ProcessEngineCoverageProperties() {
```

Comment:
This is very verbose, why not instantiating a data class? To make it usable from java (the have no named properties), the builder pattern is nice... https://stackoverflow.com/questions/36140791/how-to-implement-builder-pattern-in-kotlin

Reply:
Check. Now the new data class can be instantiated through a builder class